staging/prometheus-operator/templates/prometheus/clusterrole.yaml

```diff
@@ -25,6 +25,14 @@ rules:
   - endpoints
   - pods
   verbs: ["get", "list", "watch"]
+- apiGroups: [""]
```

Comment:
While we have an opportunity to, could we add the context about why we need these permissions in a comment here? It could be helpful for future readers to be able to tell exactly why we need these perms at a glance just by looking at the helm chart.